contracts/0.8.25/vaults/predeposit_guarantee/PredepositGuarantee.sol

```diff
@@ -547,24 +508,16 @@ contract PredepositGuarantee is IPredepositGuarantee, CLProofVerifier, PausableU
     }
 
     /**
-     * @notice returns locked ether to the staking vault owner if validator's WC were proven invalid
+     * @notice returns locked ether to the staking vault if validator's WC were proven invalid
      * @param _validatorPubkey to take locked PREDEPOSIT_AMOUNT ether from
-     * @param _recipient address to transfer PREDEPOSIT_AMOUNT ether to
-     * @dev can only be called by owner of vault that had deposited to disproven validator
      */
-    function compensateDisprovenPredeposit(
-        bytes calldata _validatorPubkey,
-        address _recipient
-    ) public whenResumed returns (uint256) {
+    function compensateDisprovenPredeposit(bytes calldata _validatorPubkey) public whenResumed returns (uint256) {
```

Comment:
let's merge the method with `proveInvalidValidatorWC`, reducing cardinality of validator stages set